.github/workflows/backend.yml

```diff
@@ -107,6 +107,57 @@ jobs:
       - name: Handle artifacts
         uses: ./.github/actions/artifacts
 
+  backend-test-in-silo-modes:
+    needs: files-changed
+    if: needs.files-changed.outputs.backend == 'true'
+      && (
+            contains(github.head_ref, '/hc-')
+         || contains(github.head_ref, 'hybrid-cloud')
+         )
+    continue-on-error: true
```

Comment:
hmmm I'm not sure if this does the right thing -- since it's only conditionally run I'd just remove it

Reply:
My understanding is that continue-on-error on jobs (not steps) applies to matrix runs -- if a single matrix run fails, it continues the others.  https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstrategyfail-fast

Reply:
that's the `fail-fast` setting `strategy: matrix` -- which is unrelated to `continue-on-error` (which marks the job red or green) -- you _don't_ want `continue-on-error` because then you won't be able to know when the job goes red